### PR TITLE
Igor admin restrictions

### PR DESCRIPTION
**Pull request by aherna**
Allows Admins to make extends without having the reservation to be in the extend window

### CODE REVIEW

File: src/igor/extend.go [hunk 1]
```diff
@@ -57,10 +57,13 @@ func runExtend(cmd *Command, args []string) {
 	}
 
 	user, err := getUser()
+
```
Comment (csymonds) [inline src/igor/extend.go]: @jcrussell : I'm not trying to nit-pick, but is it typical to have an empty line between a call and checking the err from the call?

Comment (aherna) [reply src/igor/extend.go]: we added that line so we can test my no-ouo settings. do you want me to repush?

Comment (jcrussell) [reply src/igor/extend.go]: I don't put a newline.

Comment (aherna) [reply src/igor/extend.go]: changed

File: src/igor/extend.go [hunk 2]
```diff
@@ -57,10 +57,13 @@ func runExtend(cmd *Command, args []string) {
...
 	if err != nil {
 		log.Fatalln("cannot determine current user", err)
 	}
 
+	log.Debug("Running Extend as %v", user.Username)
+
```
Comment (csymonds) [inline src/igor/extend.go]: @jcrussell This one?

File: src/igor/extend.go (hunk 1 above)
Comment (jcrussell) [reply src/igor/extend.go]: @csymonds this one.

**Comment by csymonds** [comment]
Having issues with getting the newline commit back in; starting a new branch and pr.

**Pull request closed without merging**